streamlit_template/README.md

```diff
@@ -123,14 +175,59 @@ as you see fit.
 - Once your EC2 product's `status` is set to `Available`, click it and navigate to the _Events_ tab.
 - Click the URL next to `ConnectionURI` to launch a shell session in your instance.
 - Navigate to your home directory (`cd ~`).
-- Clone your repository in your desired working directory.
-- Create your `secrets.toml` file again. The Docker image of your Streamlit application will not have the `secrets.toml` for security reasons.
-- Build your Docker image from the Dockerfile in the repository
+- Clone your repository in your desired working directory. Example:
+  ```
+  git clone https://github.com/<your-username>/snowflake.git
+  ```
+  Replace `<your-username>` with the user that the forked repository is under.
+- Create your `secrets.toml` file again, and make sure it's located under the `.streamlit/` folder. By default, the instance should already have `vi` available to use as an editor.
```

Comment:
`and make sure it's located under the `.streamlit/` folder`

In this case it doesn't actually need to exist here. It only needs to be available in the current working directory where you run the `docker run` command